### ISSUE TITLE
Getting "no editable content region" error in page edit mode

### ISSUE THREAD

**Issue by shchukax** [microweber/microweber#386]
I downloaded 1.0.7 from the main website's link and installed it with no problem.  However when I log in to admin console and go to edi the home page, I see message _"no editable content region found in file index.php (default template)"_ (see screenshot).  This is odd, as the correct `edit` class is present in the file.  I haven't been able to solve it, even by copying the file from another installation, which works correctly.  It's worth noting that live edit works fine.

![image](https://cloud.githubusercontent.com/assets/9897788/21383098/ee08daaa-c75b-11e6-8640-dfdcfb1ff210.png)

**Comment by peter-mw** [comment]
hi,

this is fixed in the 1.0.8 version which will be released soon


if you want to fix if for your current template you will need to add `field` and `rel` attributes to the edit region

for example 

 
```html
<div class="edit" field="content" rel="content">
my text
</div>
```

**Comment by shchukax** [comment]
Ah, ok, this fixed it. Interestingly, in the _lightness_ theme, that div tag was

    <div class="edit" field="lightness_content" rel="content">
